src/app/init.ts

```diff
@@ -18,5 +19,26 @@ if (userDataPath) {
     app.setPath('userData', userDataPath);
 }
 
+// Setting the env path child_process issue https://github.com/electron/electron/issues/7688
+(async () => {
+    try {
+        const paths = await shellPath();
+        if (paths) {
+            return process.env.PATH = paths;
+        }
+        process.env.PATH = [
```

Comment:
is this applicable only for macOS? If so, let's add the check

Reply:
Done 👍